src/engine/database/mod.rs

```diff
@@ -65,23 +65,23 @@ pub enum DatabasePool {
 
 impl DatabasePool {
     #[cfg(feature = "neo4j")]
-    pub(crate) fn neo4j(&self) -> Result<&bb8::Pool<bb8_bolt::BoltConnectionManager>, Error> {
+    pub fn neo4j(&self) -> Result<&bb8::Pool<bb8_bolt::BoltConnectionManager>, Error> {
```

Comment:
Warpgrapher should be passing a usable database client to extension points, custom endpoints, and custom resolvers that need one. If there's something broken in doing so, I think it would be best to fix that problem so that it works, if we can. I don't think we want to expose the pool creation functions and have people create their own database pools in extensions and custom resolvers.

Reply:
I don't think those function create new pools. I think they just return a database-specific representation of the database pool so that the user can get a client from the pool. Is that incorrect?

Reply:
You are totally correct. I totally misread where in the code that was. Let us never speak of this again...